@codex please fix

```
Run if [ "macOS" = "macOS" ]; then
Warning: Developer ID certificate and identity are not fully configured; using ad-hoc signing for preview/dev-only macOS artifacts.

> token-place-desktop-tauri@0.1.0 tauri
> tauri build --target aarch64-apple-darwin --bundles app

        Info Looking up installed tauri packages to check mismatched versions...
     Running beforeBuildCommand `npm run build`

> token-place-desktop-tauri@0.1.0 build
> vite build

vite v5.4.21 building for production...
transforming...
✓ 34 modules transformed.
rendering chunks...
computing gzip size...
dist/index.html                  0.33 kB │ gzip:  0.24 kB
dist/assets/index-ovZlYPFS.js  153.05 kB │ gzip: 48.88 kB
✓ built in 427ms
    Updating crates.io index
   Compiling token-place-desktop-tauri v0.1.0 (/Users/runner/work/token.place/token.place/desktop-tauri/src-tauri)
warning: struct `RelayEnvelope` is never constructed
  --> src/forward.rs:12:12
   |
12 | pub struct RelayEnvelope {
   |            ^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` (part of `#[warn(unused)]`) on by default

warning: function `assemble_relay_envelope` is never used
  --> src/forward.rs:26:8
   |
26 | pub fn assemble_relay_envelope(
   |        ^^^^^^^^^^^^^^^^^^^^^^^

warning: function `generate_rsa_keypair_pem` is never used
 --> src/keygen.rs:6:8
  |
6 | pub fn generate_rsa_keypair_pem() -> anyhow::Result<(String, String)> {
  |        ^^^^^^^^^^^^^^^^^^^^^^^^

warning: function `generate_rsa_keypair_b64` is never used
  --> src/keygen.rs:15:8
   |
15 | pub fn generate_rsa_keypair_b64() -> anyhow::Result<(String, String)> {
   |        ^^^^^^^^^^^^^^^^^^^^^^^^

warning: `token-place-desktop-tauri` (bin "token-place-desktop-tauri") generated 4 warnings
    Finished `release` profile [optimized] target(s) in 35.09s
       Built application at: /Users/runner/work/token.place/token.place/desktop-tauri/src-tauri/target/aarch64-apple-darwin/release/token-place-desktop-tauri
    Bundling token.place desktop.app (/Users/runner/work/token.place/token.place/desktop-tauri/src-tauri/target/aarch64-apple-darwin/release/bundle/macos/token.place desktop.app)
     Signing with identity ""
Signing with identity ""
Signing /Users/runner/work/token.place/token.place/desktop-tauri/src-tauri/target/aarch64-apple-darwin/release/bundle/macos/token.place desktop.app/Contents/MacOS/token-place-desktop-tauri
error: The specified item could not be found in the keychain.
failed to bundle project failed codesign application: failed to run command codesign: failed to sign app
       Error failed to bundle project failed codesign application: failed to run command codesign: failed to sign app
Error: Process completed with exit code 1.

```

new codex task, not a r/e/v/i/e/w task

--- a/.github/workflows/desktop-release.yml
+++ b/.github/workflows/desktop-release.yml
@@ -212,13 +212,15 @@ jobs:
 
       - name: Build Tauri bundles
         shell: bash
-        env:
-          APPLE_SIGNING_IDENTITY: ${{ secrets.APPLE_SIGNING_IDENTITY }}
-          APPLE_CERTIFICATE_P12_BASE64: ${{ secrets.APPLE_CERTIFICATE_P12_BASE64 }}
-          APPLE_CERTIFICATE_PASSWORD: ${{ secrets.APPLE_CERTIFICATE_PASSWORD }}
-          APPLE_KEYCHAIN_PASSWORD: ${{ secrets.APPLE_KEYCHAIN_PASSWORD }}
         run: |
           if [ "${{ runner.os }}" = "macOS" ]; then
+            APPLE_SIGNING_IDENTITY="${{ secrets.APPLE_SIGNING_IDENTITY }}"
+            APPLE_CERTIFICATE_P12_BASE64="${{ secrets.APPLE_CERTIFICATE_P12_BASE64 }}"
+            APPLE_CERTIFICATE_PASSWORD="${{ secrets.APPLE_CERTIFICATE_PASSWORD }}"
+            APPLE_KEYCHAIN_PASSWORD="${{ secrets.APPLE_KEYCHAIN_PASSWORD }}"
+            # Prevent tauri-bundler from auto-reading an empty APPLE_SIGNING_IDENTITY
+            # from the job environment, which triggers a failing `codesign -s ""` path.
+            unset APPLE_SIGNING_IDENTITY
             if [ -n "${APPLE_SIGNING_IDENTITY:-}" ] && [ -n "${APPLE_CERTIFICATE_P12_BASE64:-}" ] && [ -n "${APPLE_CERTIFICATE_PASSWORD:-}" ]; then
               KEYCHAIN_PATH="$RUNNER_TEMP/codesign.keychain-db"
               KEYCHAIN_PASSWORD="${APPLE_KEYCHAIN_PASSWORD:-temp_keychain_password}"

--- a/.github/workflows/desktop-release.yml
+++ b/.github/workflows/desktop-release.yml
@@ -214,14 +214,14 @@ jobs:
         shell: bash
         run: |
           if [ "${{ runner.os }}" = "macOS" ]; then
-            APPLE_SIGNING_IDENTITY="${{ secrets.APPLE_SIGNING_IDENTITY }}"
+            CONFIGURED_APPLE_SIGNING_IDENTITY="${{ secrets.APPLE_SIGNING_IDENTITY }}"
             APPLE_CERTIFICATE_P12_BASE64="${{ secrets.APPLE_CERTIFICATE_P12_BASE64 }}"
             APPLE_CERTIFICATE_PASSWORD="${{ secrets.APPLE_CERTIFICATE_PASSWORD }}"
             APPLE_KEYCHAIN_PASSWORD="${{ secrets.APPLE_KEYCHAIN_PASSWORD }}"
             # Prevent tauri-bundler from auto-reading an empty APPLE_SIGNING_IDENTITY
             # from the job environment, which triggers a failing `codesign -s ""` path.
             unset APPLE_SIGNING_IDENTITY
-            if [ -n "${APPLE_SIGNING_IDENTITY:-}" ] && [ -n "${APPLE_CERTIFICATE_P12_BASE64:-}" ] && [ -n "${APPLE_CERTIFICATE_PASSWORD:-}" ]; then
+            if [ -n "${CONFIGURED_APPLE_SIGNING_IDENTITY:-}" ] && [ -n "${APPLE_CERTIFICATE_P12_BASE64:-}" ] && [ -n "${APPLE_CERTIFICATE_PASSWORD:-}" ]; then
               KEYCHAIN_PATH="$RUNNER_TEMP/codesign.keychain-db"
               KEYCHAIN_PASSWORD="${APPLE_KEYCHAIN_PASSWORD:-temp_keychain_password}"
               CERT_PATH="$RUNNER_TEMP/apple-signing-cert.p12"
@@ -232,7 +232,7 @@ jobs:
               security import "$CERT_PATH" -k "$KEYCHAIN_PATH" -P "$APPLE_CERTIFICATE_PASSWORD" -T /usr/bin/codesign
               security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PASSWORD" "$KEYCHAIN_PATH"
               security list-keychains -d user -s "$KEYCHAIN_PATH" $(security list-keychains -d user | tr -d '"')
-              export TAURI_BUNDLE_MACOS_SIGNING_IDENTITY="$APPLE_SIGNING_IDENTITY"
+              export TAURI_BUNDLE_MACOS_SIGNING_IDENTITY="$CONFIGURED_APPLE_SIGNING_IDENTITY"
               export APPLE_CERTIFICATE_IMPORTED=1
             else
               echo "::warning::Developer ID certificate and identity are not fully configured; using ad-hoc signing for preview/dev-only macOS artifacts."

--- a/.github/workflows/desktop-release.yml
+++ b/.github/workflows/desktop-release.yml
@@ -233,10 +233,12 @@ jobs:
               security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PASSWORD" "$KEYCHAIN_PATH"
               security list-keychains -d user -s "$KEYCHAIN_PATH" $(security list-keychains -d user | tr -d '"')
               export TAURI_BUNDLE_MACOS_SIGNING_IDENTITY="$CONFIGURED_APPLE_SIGNING_IDENTITY"
+              export APPLE_SIGNING_IDENTITY="$CONFIGURED_APPLE_SIGNING_IDENTITY"
               export APPLE_CERTIFICATE_IMPORTED=1
             else
               echo "::warning::Developer ID certificate and identity are not fully configured; using ad-hoc signing for preview/dev-only macOS artifacts."
               export TAURI_BUNDLE_MACOS_SIGNING_IDENTITY='-'
+              export APPLE_SIGNING_IDENTITY='-'
             fi
           fi
           if [ "${{ runner.os }}" = "macOS" ] && [ -n "${{ matrix.tauri_target }}" ]; then

